src/main/java/uk/gov/cshr/service/security/IdentityService.java

```diff
@@ -93,6 +93,7 @@ public void updatePassword(Identity identity, String password) {
         identity.setDeletionNotificationSent(false);
         identity.setPassword(passwordEncoder.encode(password));
         identityRepository.save(identity);
+        identity.setLocked(false);
```

Comment:
Move above the .save() function otherwise your setLocked(false) update wont be persisted